frontend/packages/kubevirt-plugin/src/components/modals/menu-actions-modals/delete-vm-modal.tsx

```diff
@@ -117,22 +124,31 @@ export const DeleteVMModal = withHandlePromise((props: DeleteVMModalProps) => {
                 onChange={() => setDeleteVMImport(!deleteVMImport)}
                 checked={deleteVMImport}
               />
-              Delete {VirtualMachineImportModel.label} Resource
+              {t('kubevirt-plugin~Delete {{vmiModelLabel}} Resource', {
+                vmiModelLabel: VirtualMachineImportModel.label,
```

Comment:
nit: I would choose a longer name so it is not confused with VMI

Reply:
Done!